autogpt/config/config.py

```diff
@@ -10,12 +10,16 @@
 from autogpt.singleton import Singleton
 
 
+
 class Config(metaclass=Singleton):
     """
     Configuration class to store the state of bools for different scripts access.
     """
 
     def __init__(self) -> None:
+        """Load here to avoid circular imports."""
```

Comment:
do this after the """initialize the config class""" as it will replace the docstring for the function which is sub optimal

Reply:
bd4637a